products/template/build_scripts/export.py

```diff
@@ -24,23 +24,25 @@
     ],
 }
 
-assert PRODUCT_METADATA_REPO_PATH
-org_metadata = OrgMetadata.from_path(Path(PRODUCT_METADATA_REPO_PATH))
 
+def generate_data_dictionaries():
+    dataset_metadata_path = PRODUCT_PATH / "data_dictionary.yml"
+
+    org_metadata_path = download_repo("product-metadata", PRODUCT_PATH)
+    org_metadata = OrgMetadata.from_path(org_metadata_path)
+
+    metadata = org_metadata.product("template_db").dataset("template_db")
+    metadata.write_to_yaml(dataset_metadata_path)
 
-def generate_metadata():
-    dataset_metadata_yml = product_metadata.download(
-        "template_db", PRODUCT_PATH / "data_dictionary.yaml", dataset="template_db"
-    )
     html_path = generate_metadata_assets.generate_html_from_yaml(
```

Comment:
Seems like it'd be a lot easier to just pass the OrgMetadata to whatever method generates the assets, yeah? In general, I've been defaulting to only requiring a path for typer commands.